SU-SWS/stanford_profile_helper
# D8CORE-4097: Fixup the specificity problem for the local footer color

# READY FOR REVIEW

# Summary
- Needed to increase specificity for the footer color

# Needed By (Date)
- 4/30

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Clear caches. 
3. Make sure you turn on the intranet.
4. Verify the local footer color is #eaeaea.
5. Verify the body background is still $su-color-fog.
6. Verify the super footer background is still white. 

# Affected Projects or Products
- Intranet onlyl

# Associated Issues and/or People
- D8CORE-4097

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
